.circleci/config.yml

```diff
@@ -0,0 +1,288 @@
+      nodejs:
+        <<: *nodejs_enum
+    <<: *defaults
+    docker:
+      - image: circleci/node:<< parameters.nodejs >>
+    steps:
+      - attach_project_repo
+      - checkout
+      - restore_build_cache:
+          suffix: << parameters.nodejs >>
+      - run_npm_ci
+      - save_build_cache:
+          suffix: << parameters.nodejs >>
+      - run_npm_build:
+          node_env: test
+      - unless:
+          condition:
+            equal: [*nodejs_current, << parameters.nodejs >>]
+          steps:
+            # `flow check` does a static analysis, no need to run more than once
+            # per worflow run.
+            - run:
+                name: Turn off flow checks on non current nodejs version
+                command: export CI_SKIP_FLOWCHECK=y
+      ## Skip code coverage and the additional legacy bundling tests on jobs
+      ## running on the next nodejs versions.
+      - unless:
+          condition:
+            equal: [*nodejs_current, << parameters.nodejs >>]
+          steps:
+            - run:
+                name: run linting checks and unit tests
+                command: npm run test
+            - run_functional_tests
+      ## Steps only executed in jobs running on the current nodejs version.
+      - when:
+          condition:
+            equal: [*nodejs_current, << parameters.nodejs >>]
+          steps:
+            - run:
+                name: run linting check and unit tests with coverage
+                command: npm run test-coverage
+            - store_artifacts:
+                path: coverage
+            - run:
+                name: publish coverage data
+                command: npm run publish-coverage
+            - run_functional_tests:
+                legacy_bundling: true
+            - persist_project_repo
+            - audit_deps
+            - lint_commit_conventions
+
+  test-windows:
+    <<: *defaults
+    executor:
+      name: win/vs2019
+      shell: cmd.exe
+    steps:
+      - checkout
+      - restore_build_cache:
+          suffix: windows
+      - run_npm_ci
+      - save_build_cache:
+          suffix: windows
+      - run_npm_build:
+          node_env: test
+      - run:
+          name: run linting checks and unit tests
+          command: npm run test
+          environment:
+            # `flow check` does a static analysis, no need to run more than once
+            # per worflow run.
+            CI_SKIP_FLOWCHECK: y
+      - run_functional_tests:
+          retry_once: "y"
+
+  release-tag:
+    parameters:
+      nodejs:
+        <<: *nodejs_enum
+    <<: *defaults
+    docker:
+      - image: circleci/node:<< parameters.nodejs >>
+    steps:
+      - attach_project_repo
+      - run_npm_build:
+          node_env: production
+      - run:
+          name: npm registry auth
+          command: echo '//registry.npmjs.org/:_authToken=${NPM_TOKEN}' > .npmrc
+      - run:
+          name: npm registry publish
+          command: npm publish --access=public
```

Comment:
Isn't `public` the default access?

Reply:
yep, is not for scoped packages, which I used to test this command, not needed in this case, but I guess it shouldn't harm neither